tools/socktap/autotalks.hpp

```diff
@@ -0,0 +1,63 @@
+#ifndef AUTOTALKS_HPP_
+#define AUTOTALKS_HPP_
+
+#include <memory>
+#include <vanetza/net/mac_address.hpp>
+#include <vanetza/access/data_request.hpp>
+#include <vanetza/net/chunk_packet.hpp>
+#include <vanetza/net/ethernet_header.hpp>
+#include <vanetza/net/cohesive_packet.hpp>
+#include "stddef.h"
+#include "atlk/sdk.h"
+#include "atlk/v2x_service.h"
+#include "autotalks_link.hpp"
+
+/*
+ * Device initialization.
+ */
+int v2x_device_init(void);
```

Comment:
This function is called `v2x_device_init` but the corresponding deinitialization function is prefixed "autotalks". This is strange.

Reply:
Renamed to `autotalks_device_init()`